src/styles/blocks/stars--1.css

```diff
@@ -0,0 +1,44 @@
+.stars--1 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 22px;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead, style all elements using `.classes`. For example, use a class like '.stars__star--active' for the first child instead of using the `:nth-child` pseudo-class.